src/qt/rpcconsole.cpp

```diff
@@ -706,6 +706,14 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
 
         // create ban table context menu
         banTableContextMenu = new QMenu(this);
+        /*: Context menu action to copy the IP/Netmask of a banned peer.
+            IP/Netmask is the combination of a peer's IP address and it's Netmask.
+            For IP Address see: https://en.wikipedia.org/wiki/IP_address
+            For NetMask see: https://wiki.teltonika-networks.com/view/What_is_a_Netmask%3F */
```

Comment:
I'd avoid mentioning any links to proprietary sites in the Bitcoin Core codebase.

Maybe, just skip last two lines of the comment?

Reply:
wikipedia or the other website? we already link to wikipedia articles to help with translations: https://github.com/bitcoin-core/gui/blob/d3203a99d886177eee9d1f9cd8411e215118a4e6/src/qt/addressbookpage.cpp#L285

Reply:
At least, drop `https://wiki.teltonika-networks.com`, please.